docs/source/getting_started/contributing_guide.rst

```diff
@@ -138,6 +138,26 @@ Git
       review.
     - Branches are merged back into ``develop``.
 
+GitHub
+------------------------------------------------------------------------------
+
+.. note::
+
+    Internal Staff Only
+
+- Mention a user (using the ``@`` symbol) when an issue is relevant to them.
+- Only assign yourself to an issue, when you are actively working on an issue.
+- When we will tag you to review specific pull requests, you can either
+  approve, request changes, or just leave comments.
+- A pull request requires one review approval to be merged.
+- If multiple people are tagged as reviewers, we only need one review (unless
+  otherwise specified).
+  We tag all those suitable for review.
```

Comment:
maybe add that "for content changes, we ask that at least one member from each of the content and technical teams reviews the PR" or something of the like (just to give an explicit example of where we want more than one reviewer)